roles/pulp_api/tasks/main.yml

```diff
@@ -62,17 +62,30 @@
   when:
     - (ansible_version.major > 2) or (ansible_version.major == 2 and ansible_version.minor >= 9)
 
+- name: Get database config
+  command: "cat {{ __pulp_db_fields_key_path }}"
+  delegate_to: "{{ __pulp_real_sole_database_config_host }}"
+  register: pulp_db_fields_key
```

Comment:
This variable name is already used for the filepath to import from.

Not the contents of it.

Reply:
OK, but it is a private variable. So we should put 2 underscores at the beginning.